app/forms/changeGroupLead/NewGroupLeadEnterContactAddressFormProvider.scala

```diff
@@ -85,6 +82,13 @@ class NewGroupLeadEnterContactAddressFormProvider {
       .verifying(invalidCharacterKey, _.matches(addressLineRegExp))
   }
 
+  private def countryCodeValidation = {
+    optional(play.api.data.Forms.text)
+      .verifying(countryCodeRequiredKey, _.isDefined)
+      .transform[String](_.get, Some(_))
+      .verifying(countryCodeRequiredKey, _.trim.nonEmpty)
```

Comment:
seperate pr/ add to work flowy but can the form check it is one of the country codes that we provided from the list